Formula/postgresql.rb

```diff
@@ -98,6 +98,9 @@ def post_install
     (var/"postgres").mkpath
     unless File.exist? "#{var}/postgres/PG_VERSION"
       system "#{bin}/initdb", "#{var}/postgres"
+      system "brew", "services", "start", "postgresql"
```

Comment:
@MikeMcQuaid I'm sorry, but I'm unfamiliar with adding to the `caveats`. Does it mean adding a warning text after installation has been completed?